reproman/interface/execute.py

```diff
@@ -132,6 +133,9 @@ def __init__(self, resource, command, cmd_args,
         self.local_trace_dir = op.join(local_cache_dir, 'traces', self.exec_id)
         self.local_tracer_gz = op.join(self.local_tracer_dir,
                                        "reproman_trace.gz")
+        self.local_docker_shim = op.join(op.dirname(op.realpath(__file__)),
+            "docker.shim")
```

Comment:
nitpick: Not indenting to match the opening parenthesis on the previous line will make most pep8 linters light up, I think.  (This applies to a few spots in this series.)